src/Forms/Shared/Samples/Layers/DisplayDimensions/DisplayDimensions.xaml.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright 2022 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+// language governing permissions and limitations under the License.
+
+using ArcGISRuntime.Samples.Managers;
+using Esri.ArcGISRuntime.Mapping;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace ArcGISRuntimeXamarin.Samples.DisplayDimensions
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        name: "Display dimensions",
+        category: "Layers",
+        description: "Display dimension features from a mobile map package.",
+        instructions: "When the sample loads, it will automatically display the map containing dimension features from the mobile map package. The name of the dimension layer containing the dimension features is displayed in the controls box. Control the visibility of the dimension layer with the \"Dimension Layer visibility\" check box, and apply a definition expression to show dimensions of greater than or equal to 450m in length using the \"Definition Expression\" checkbox.",
+        tags: new[] { "dimension", "layer", "mmpk", "mobile map package", "utility", "Featured" })]
+    [ArcGISRuntime.Samples.Shared.Attributes.OfflineData("f5ff6f5556a945bca87ca513b8729a1e")]
+    public partial class DisplayDimensions : ContentPage
+    {
+        // Hold a reference to the Dimension layer for use in event handlers
+        private DimensionLayer _dimensionLayer;
+
+        public DisplayDimensions()
+        {
+            InitializeComponent();
+            _ = Initialize();
+        }
+
+        private async Task Initialize()
+        {
+            try
+            {
+                // Load the mobile map package.
+                MobileMapPackage mobileMapPackage = new MobileMapPackage(DataManager.GetDataFolder("f5ff6f5556a945bca87ca513b8729a1e", "Edinburgh_Pylon_Dimensions.mmpk"));
```

Comment:
I recommend breaking the call to DataManager.GetDataFolder out into a separate line. Ideally it should be very clear that the mobile map package constructor is just taking a path and that `DataManager` is a sample viewer-specific tool, not part of the Runtime API. Customers have been confused by this in the past.

Reply:
e.g. something like this:

```cs
// Get the path to the map package. DataManager is a sample viewer tool, not part of ArcGIS Runtime.
var dataPath = DataManager.GetDataFolder("f5ff6f5556a945bca87ca513b8729a1e", "Edinburgh_Pylon_Dimensions.mmpk");

// Load the mobile map package.
MobileMapPackage mobileMapPackage = new MobileMapPackage(dataPath);
```